examples/pipelines/windows-installer/windows-installer-pipeline.yaml

```diff
@@ -0,0 +1,320 @@
+                    <HideOnlineAccountScreens>true</HideOnlineAccountScreens>
+                    <HideWirelessSetupInOOBE>true</HideWirelessSetupInOOBE>
+                    <NetworkLocation>Home</NetworkLocation>
+                    <SkipUserOOBE>true</SkipUserOOBE>
+                    <SkipMachineOOBE>true</SkipMachineOOBE>
+                    <ProtectYourPC>3</ProtectYourPC>
+                </OOBE>
+                <UserAccounts>
+                    <LocalAccounts>
+                        <LocalAccount wcm:action="add">
+                            <Password>
+                                <Value>administrator</Value>
+                                <PlainText>true</PlainText>
+                            </Password>
+                            <Description></Description>
+                            <DisplayName>Administrator</DisplayName>
+                            <Group>Administrators</Group>
+                            <Name>Administrator</Name>
+                        </LocalAccount>
+                    </LocalAccounts>
+                </UserAccounts>
+                <RegisteredOrganization></RegisteredOrganization>
+                <RegisteredOwner>Administrator</RegisteredOwner>
+                <DisableAutoDaylightTimeSet>false</DisableAutoDaylightTimeSet>
+                <FirstLogonCommands>
+                    <SynchronousCommand wcm:action="add">
+                        <Description>Control Panel View</Description>
+                        <Order>1</Order>
+                        <CommandLine>reg add "HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ControlPanel" /v StartupPage /t REG_DWORD /d 1 /f</CommandLine>
+                        <RequiresUserInput>true</RequiresUserInput>
+                    </SynchronousCommand>
+                    <SynchronousCommand wcm:action="add">
+                        <Order>2</Order>
+                        <Description>Control Panel Icon Size</Description>
+                        <RequiresUserInput>false</RequiresUserInput>
+                        <CommandLine>reg add "HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ControlPanel" /v AllItemsIconView /t REG_DWORD /d 0 /f</CommandLine>
+                    </SynchronousCommand>
+                    <SynchronousCommand wcm:action="add">
+                        <Order>3</Order>
+                        <RequiresUserInput>false</RequiresUserInput>
+                        <CommandLine>cmd /C wmic useraccount where name="Administrator" set PasswordExpires=false</CommandLine>
+                        <Description>Password Never Expires</Description>
+                    </SynchronousCommand>
+                    <SynchronousCommand wcm:action="add">
+                        <Order>4</Order>
+                        <RequiresUserInput>false</RequiresUserInput>
+                        <CommandLine>cmd /c shutdown /s /f /t 10</CommandLine>
```

Comment:
I'm missing here the sysprep command.
you mention it in the readme  but no reference to it

Reply:
It should be run automatically by the windows installer when it locates `autounattend.xml`